apps/web/playwright/profile.e2e.ts

```diff
@@ -199,18 +197,18 @@ test.describe("Update Profile", () => {
   }) => {
     await test.step("the user receives the correct invitation link", async () => {
       await page.waitForLoadState("networkidle");
-      const verificationToken = await prisma.verificationToken.findFirst({
-        where: {
-          identifier: secondaryEmail,
-        },
-      });
```

Comment:
Not quite sure why but verificationToken is undefined here. So the inviteLink from `expectInvitationEmailToBeReceived` has correct link and token, but string in toEqual part is wrong